### PR TITLE
tmux: add commands

### PR DESCRIPTION
**Pull request by TrevCan**
- add terminology of windows, panes, and sessions
- add cycle through windows command
- add cycle through panes command
- add go to most recent pane command

- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/tmux.md [hunk 1]
```diff
@@ -19,18 +19,41 @@
...
+- Detach from the current session:
 
 `Ctrl-B d`
 
-- Create a new window (inside a tmux session):
+- Terminology (from `man tmux`):
+  -   Session: collection of pseudo terminals managed by tmux
+  -   Window: occupies the entire screen, can be split into panes (rectangles)
+  -   Pane: one pseudo terminal
+
```
Comment (CleanMachine1) [inline pages/common/tmux.md:32]: Not needed or a part of our syntax

File: pages/common/tmux.md [hunk 2]
```diff
@@ -19,18 +19,41 @@
 
 `tmux attach`
 
-- Detach from the current session (inside a tmux session):
+INSIDE A TMUX SESSION:
```
Comment (CleanMachine1) [inline pages/common/tmux.md:22]: Every command in TLDR has the potential to be randomized, so each command will need `While running tmux.........`

File: pages/common/tmux.md [hunk 3]
```diff
@@ -19,18 +19,41 @@
...
+  -   Window: occupies the entire screen, can be split into panes (rectangles)
+  -   Pane: one pseudo terminal
+
+- Create a new window:
 
 `Ctrl-B c`
 
-- Switch between sessions and windows (inside a tmux session):
+- Cycle through windows:
+
```
Comment (CleanMachine1) [inline pages/common/tmux.md:38]:
```suggestion
- While running tmux, cycle through windows:

```
please follow the syntax used here.

I have never used tmux, does it use caps for its name, like `TMux`

Comment (mfrw) [reply pages/common/tmux.md:38]: > please follow the syntax used here.
> 
> I have never used tmux, does it use caps for its name, like `TMux`

I have always seen it as `tmux`

Comment (marchersimon) [reply pages/common/tmux.md:38]: > I have never used tmux, does it use caps for its name, like `TMux`

No, it's just `tmux`

**Comment by marchersimon** [comment]
Hey @TrevCan, thanks for you input. Unfortunately, we can't apply most of your suggestions, since:
* We have a strict syntax for examples:
```
- One-line example description:

`example`
```
* Our pages can have at most 8 examples (and `tmux already has 8`)

Bigger cheatsheets with lots of shortcuts are important of course, but that's not the goal of `tldr`

File: pages/common/tmux.md (hunk 3 above)
Comment (CleanMachine1) [reply pages/common/tmux.md:38]: Ok

**Comment by TrevCan** [comment]
> Hey @TrevCan, thanks for you input. Unfortunately, we can't apply most of your suggestions, since:
> 
> * We have a strict syntax for examples:
> 
> ```
> - One-line example description:
> 
> `example`
> ```
> 
> * Our pages can have at most 8 examples (and `tmux already has 8`)
> 
> Bigger cheatsheets with lots of shortcuts are important of course, but that's not the goal of `tldr`

Hey @marchersimon, I understand. I'll close the PR, I thought there was still enough room for some commands.

**Comment by marchersimon** [comment]
If you're searching for inspiration for contributing something else, chech out <https://github.com/tldr-pages/tldr/issues>.

**Pull request closed without merging**